.eslintrc.js

```diff
@@ -3,7 +3,20 @@ const { monorepo } = require('@ulixee/repo-tools/eslint');
 module.exports = monorepo(__dirname);
 
 module.exports.ignorePatterns.push('DomExtractor.js');
-module.exports.overrides.push({
-  files: ['browser-profiler/**/*', 'double-agent-stacks/**/*'],
-  rules: { 'no-console': 'off' },
-});
+module.exports.overrides.push(
+  {
+    files: ['browser-profiler/**/*', 'double-agent-stacks/**/*'],
+    rules: {
+      'no-console': 'off',
+    },
+  },
+  {
+    files: ['**/*.test.ts'],
```

Comment:
Is this going to allow focused tests to slip in?

Reply:
No this only adds an extra rules that allows using `testIfNotOnGithubWindows` or `testIfNotOnGithubMac` without linter complaining because it doesn't recognize it as a test